### PR TITLE
Feature/0005 docker

### PR DESCRIPTION
**Pull request by ajax-torbin-v**

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+  mongo:
+    image: mongo:latest
+    container_name: mongo
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=${MONGO_USER}
+      - MONGO_INITDB_ROOT_PASSWORD=${MONGO_PASS}
+    env_file: .env
+    volumes:
+      - ./init-mongo.sh:/docker-entrypoint-initdb.d/init-mongo.sh
```
Comment (ajax-malov-n) [inline docker-compose.yaml]: include script

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Use a base image with JDK and Gradle
+FROM openjdk:17-jdk-slim
+
+# Set the working directory for the runtime environment
+WORKDIR /app
+
+# Copy the compiled JAR file from the build stage
+COPY ./build/libs/delivery-0.0.1-SNAPSHOT.jar app.jar
```
Comment (ajax-marchenko-s) [inline Dockerfile:8]: you can use `delivery-*.jar` instead of specifying the version

**Review by ajax-marchenko-s — COMMENTED** [review]
Good work

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Use a base image with JDK and Gradle
+FROM openjdk:17-jdk-slim
```
Comment (ajax-gnatiuk-s) [inline Dockerfile]: `amazoncorretto`? 😁

**Review by ajax-samondros-b — APPROVED** [review]

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Use a base image with JDK and Gradle
+FROM amazoncorretto:22-alpine3.17-jdk
```
Comment (ajax-shulepa-a) [inline Dockerfile]:
```suggestion
FROM amazoncorretto:22
```

File: Dockerfile (hunk 2 above)
Comment (ajax-shulepa-a) [reply Dockerfile:8]: why is it resolved without any reply or changes?

File: k8s/config/mongo-express-configmap.yaml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: mongo-express-configmap
+data:
+  mongodb-url: "mongodb://root:12345@mongodb-service:27017/?authSource=admin"
```
Comment (ajax-shulepa-a) [inline k8s/config/mongo-express-configmap.yaml]: https://stackoverflow.com/a/54205836

File: k8s/config/mongo-express-deployment.yaml [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+              valueFrom:
+                secretKeyRef:
+                  name: mongodb-secret
+                  key: mongodb-root-username
+            - name: ME_CONFIG_MONGODB_ADMINPASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: mongodb-secret
+                  key: mongodb-root-password
+            - name: ME_CONFIG_BASICAUTH_USERNAME
```
Comment (ajax-shulepa-a) [inline k8s/config/mongo-express-deployment.yaml]: Does mongodb-express actually use these properties?

**Review by ajax-shulepa-a — CHANGES_REQUESTED** [review]

**Pull request merged**